.snippets/text/products/reference/ntt/overview/ntt-timeline.json

```
@@ -0,0 +1,17 @@
+[
+    {
+        "title": "[Get Started with NTT](/docs/products/native-token-transfers/get-started/)",
+        "content": "Start your journey with Wormhole's NTT by learning to install its CLI, deploy test tokens, and set up it's initial structure for seamless token movement.",
+        "icon": ":fontawesome-solid-1:"
+    },
+    {
+        "title": "Deploy NTT",
+        "content": "If seeking guidance deploying native token transfers across blockchains like EVM-compatible chains and Solana here are some resources.  \n**→** [Deploy to EVM](/docs/products/native-token-transfers/guides/deploy-to-evm/)  \n\n**→** [Deploy to EVM Chains via Launchpad](/docs/products/native-token-transfers/guides/evm-launchpad/)   \n**→** [Deploy to Solana](/docs/products/native-token-transfers/guides/deploy-to-solana/)",
```

```suggestion
        "content": "Use these resources to deploy NTT across supported chains: <br><br>**→** [Deploy to EVM](/docs/products/native-token-transfers/guides/deploy-to-evm/)  \n\n**→** [Deploy to EVM Chains via Launchpad](/docs/products/native-token-transfers/guides/evm-launchpad/)   \n**→** [Deploy to Solana](/docs/products/native-token-transfers/guides/deploy-to-solana/)",
```

Sorry haha sometimes I ask for things and then this happens. Just something to make it more concise. Also just FYI, sometimes it's easier to use `<br>` to force line breaks with the timelines